cirq-core/cirq/ops/two_qubit_diagonal_gate.py

```diff
@@ -31,12 +31,28 @@
 
 @value.value_equality()
 class TwoQubitDiagonalGate(raw_types.Gate):
-    """A gate given by a diagonal 4\\times 4 matrix."""
+    r"""A two qubit gate whose unitary is a diagonal $4 \times 4$ matrix.
+
+    This gate's off-diagonal elements are zero and its on diagonal
```

Comment:
I think this should say "on-diagonal" or just "diagonal".

(Applies below on l.55, too.)